core/src/main/java/fr/sncf/osrd/railjson/schema/rollingstock/RJSRollingStock.java

```diff
@@ -67,10 +66,16 @@ public class RJSRollingStock implements Identified {
     public double comfortAcceleration = Double.NaN;
 
     /**
-     * The braking deceleration coefficient can be the max or constant (depends on type field).
+     * The braking deceleration coefficient used for timetable calculation.
      */
-    public RJSGamma gamma = null;
+    @Json(name = "timetable_gamma")
+    public double timetableGamma = Double.NaN;
 
+    /**
+     * The max braking force of the rolling stock.
+     */
+    @Json(name = "max_braking_force")
+    public double maxBrakingForce = Double.NaN;
```

Comment:
These changes will probably need API changes

Reply:
Yep i guess ! I wanted to check this with you guys before merging 😁